main.js

```diff
@@ -412,7 +420,7 @@ class openknx extends utils.Adapter {
             const out = outpath + id.replace(inpath, "");
             this.setState(out, {
                 val: state.val,
-                ack: true,
+                ack: false,
```

Comment:
ioBroker ACK false means this will interpret as user input and will send as write to the KNX bus